src/renderer/components/+extensions/installed-extensions.tsx

```diff
@@ -136,10 +132,17 @@ const NonInjectedInstalledExtensions = observer(({ extensionDiscovery, extension
               <span className="title" aria-disabled={isUninstalling}>Uninstall</span>
             </MenuItem>
           </MenuActions>
-        ),
-      };
-    }), [extensions, extensionInstallationStateStore.anyUninstalling],
-  );
+        )
+      }
+    }
+  ]), []);
+
+  const table = useReactTable({
```

Comment:
I think "installed extensions" shouldn't know about technical stuff of table like `useReactTable`.